extensions/librdkafka/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ ExternalProject_Add(
                "-DRDKAFKA_BUILD_EXAMPLES=OFF"
                "-DRDKAFKA_BUILD_TESTS=OFF"
                "-DENABLE_LZ4_EXT=OFF"
+               "-DWITH_ZSTD=OFF"
```

Comment:
For the record: this can fail build in case a non-compatible version of zstd is available.
That's why it passed in our docker env, but failed on my server.